### PR TITLE
Refine 'Administrative monitors' interface

### PR DESCRIPTION
**Pull request by janfaracik**
Small PR to update the 'Administrative monitors' interface.

**Before**
image

**After**
image

**What's changed**
* Content simplified, dropped superfluous 'configuration' suffix, dropped repeated checkbox wrapper label
* Now using the shared `jenkins-badge` class
* Unused CSS removed (checked core and plugins, no usages)
* Checkbox descriptions now work in the same way as the radio descriptions, previously there was an additional margin added

### Testing done

* Works as before

### Proposed changelog entries

- Refine 'Administrative monitors' interface.

### Proposed upgrade guidelines

N/A

```[tasklist]
### Submitter checklist
- [ ] The Jira issue, if it exists, is well-described.
- [ ] The changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developers, depending on the change) and are in the imperative mood (see [examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)). Fill in the **Proposed upgrade guidelines** section only if there are breaking changes or changes that may require extra steps from users during upgrade.
- [ ] There is automated testing or an explanation as to why this change has no tests.
- [ ] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadocs, as appropriate.
- [ ] New deprecations are annotated with `@Deprecated(since = "TODO")` or `@Deprecated(forRemoval = true, since = "TODO")`, if applicable.
- [ ] New or substantially changed JavaScript is not defined inline and does not call `eval` to ease future introduction of Content Security Policy (CSP) directives (see [documentation](https://www.jenkins.io/doc/developer/security/csp/)).
- [ ] For dependency updates, there are links to external changelogs and, if possible, full differentials.
- [ ] For new APIs and extension points, there is a link to at least one consumer.
```

### Desired reviewers

@jenkinsci/sig-ux 

Before the changes are marked as `ready-for-merge`:

```[tasklist]
### Maintainer checklist
- [ ] There are at least two (2) approvals for the pull request and no outstanding requests for change.
- [ ] Conversations in the pull request are over, or it is explicit that a reviewer is not blocking the change.
- [ ] Changelog entries in the pull request title and/or **Proposed changelog entries** are accurate, human-readable, and in the imperative mood.
- [ ] Proper changelog labels are set so that the changelog can be generated automatically.
- [ ] If the change needs additional upgrade steps from users, the `upgrade-guide-needed` label is set and there is a **Proposed upgrade guidelines** section in the pull request title (see [example](https://github.com/jenkinsci/jenkins/pull/4387)).
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins.io/issues/?filter=12146)).
```

### CODE REVIEW

File: core/src/main/resources/jenkins/management/AdministrativeMonitorsConfiguration/config.groovy
Comment (timja) [inline core/src/main/resources/jenkins/management/AdministrativeMonitorsConfiguration/config.groovy:37]: any reason not to use one of the existing classes for spacing?

File: war/src/main/scss/components/_badges.scss
Comment (daniel-beck) [inline war/src/main/scss/components/_badges.scss:29]: These were just unused, right? Or what does this do?

File: core/src/main/resources/jenkins/management/AdministrativeMonitorsConfiguration/config.groovy
Comment (janfaracik) [reply core/src/main/resources/jenkins/management/AdministrativeMonitorsConfiguration/config.groovy:37]: They're too large - smallest is 1rem. Might be an idea to have small spacing classes.

**Review by timja — APPROVED** [review]
LGTM

**Review by daniel-beck — APPROVED** [review]

**Comment by timja** [comment]
/label ready-for-merge

----

This PR is now ready for merge, after ~24 hours, we will merge it if there's no negative feedback.

Thanks!

**Review by NotMyFault — APPROVED** [review]

**Pull request merged**
